backtrace-library/src/main/java/backtraceio/library/breadcrumbs/BacktraceBreadcrumbsLogger.java

```diff
@@ -0,0 +1,88 @@
+package backtraceio.library.breadcrumbs;
+
+import java.io.IOException;
+import java.util.Map;
+
+import backtraceio.library.enums.BacktraceBreadcrumbLevel;
+import backtraceio.library.enums.BacktraceBreadcrumbType;
+import backtraceio.library.logger.BacktraceLogger;
+
+public class BacktraceBreadcrumbsLogger {
```

Comment:
Hey  - can you share with us the purpose of this class? Why we can't include implementation of this class in BacktraceBreadcrumbs implementation instead? Each class should have different responsibilities - how can we define BacktraceBreadcrumbs responsibilities and how can we compare them with BacktraceBreadcrumbsLogger responsibilities?

Reply:
I can rename some classes for clarity. Only thing `BacktraceBreadcrumbsLogger` does is define our log format, then write to whatever the actual log writer is, in this case `BacktraceQueueFileHelper` but we might want to change the log writer in the future. 

That is: 
`BacktraceBreadcrumbs` high-level manager of all breadcrumb logic
`BacktraceBreadcrumbLogger` what should a breadcrumb look like?
`BacktraceQueueFileHelper` breadcrumb log writer

Does this help explain anything? Do you have any suggestions to make these relationships more clear? Maybe I can add comments?

Reply:
I have renamed to `BacktraceBreadcrumbLogManager`, I think this makes things a bit clearer.